internal/pkg/teleport/secret.go

```diff
@@ -27,6 +27,7 @@ type SecretConfig struct {
 
 func GetConfigFromSecret(ctx context.Context, ctrlClient client.Client, namespace string) (*SecretConfig, error) {
 	secret := &corev1.Secret{}
```

Comment:
Doesn't look like any information in the secret is secret now

Reply:
Right, should go to config map.